.github/workflows/python-package.yml

```diff
@@ -13,8 +13,25 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
 
+  formatting:
+    name: Check Formatting
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+    - name: Install Formatting
+      run: |
+        pip install autopep8 flake8
+    - name: Check Formatting
+      run: |
+        flake8 collada
+        autopep8 --recursive --aggressive --diff --exit-code collada/
+
+  build:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
```

Comment:
is the flake8 workflow on line 56 obsolete now? I already had some flake8 checks selected below?

Reply:
Oh totally missed that! Updated the existing check and removed my added check.